src/Components/ProductCard/ProductCard.css

```diff
@@ -0,0 +1,90 @@
+  
+  .card-ecom {
+    display: flex;
+    flex-direction: column;
+    background-color: var(--white);
+    width: 14rem;
+    box-shadow: 0.3rem 0.5rem 0.6rem 0.2rem grey;
```

Comment:
why box-shadow in rem ?
if you want to change shadow according to screen size then its fine but, if not then u should use px. bcz for properties like box-shadow and border-radius px is considered as good practice

Reply:
Yes i got it. Kept it that way on purpose.